Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -60,8 +60,6 @@ private void Update()
             gazeDirection = Camera.main.transform.forward;
 
             UpdateRaycast();
-
-            UpdateStabilizationPlane();
```

Comment:
I'm unsure why GazeManager should have any control over the stabilization plane logic. I understand that it made sense at the time to have it there since it used GazeManager's public facing properties, but seeing as there are several equally useful ways to set the stabilization plane (including the way GazeManager did it) it seems that the stabilization should be fully broken out into its own file.